src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -132,8 +132,29 @@ trait IterableOps[+A] extends Any {
   }
 
   override def toString = s"$className(${mkString(", ")})"
+
+  def sameElements[B >: A](that: Iterable[B]): Boolean = {
```

Comment:
Should this be in `IterableOps`? It doesn't really make sense for Sets and Maps with an undefined traversal order.